test/PsrLoggerAdapterTest.php

```diff
@@ -98,7 +98,9 @@ public function testPsrLogLevelsMapsToPriorities($logLevel, $priority)
         $message = 'foo';
         $context = ['bar' => 'baz'];
 
-        $logger = $this->getMock(Logger::class, ['log']);
+        $logger = $this->getMockBuilder(Logger::class)
+                ->setMethods(['log'])
+                ->getMock();
```

Comment:
Please align indents in two above lines, should be as below:
```
        $logger = $this->getMockBuilder(Logger::class)
            ->setMethods(['log'])
            ->getMock();
```

(so only 4 spaces more than in line 101).